.pipelines/e2e.yaml

```diff
@@ -0,0 +1,23 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)_e2e
+trigger:
+  branches:
+    include:
+    - ace/moe2e
```

Comment:
Not sure I follow, why is there a specific branch in the trigger?

Reply:
it's the the branch I was developing on, whoops